Buy_Nothing_Day.md

```diff
@@ -0,0 +1 @@
+#2021.11.27 Dzień bez zakupów
```

Comment:
Jak w plikach z rozszerzeniem .md damy spację po znaku hasz # to to jest nagłówek i inaczej się wyświetla w podglądzie - lepiej. To są pliki markdown (oczywiście nie wpływa to na ocenę, tak tylko informacyjnie :) )